src/reference-manual/statements.qmd

```diff
@@ -361,6 +361,38 @@ including arrays of vectors and matrices.   For container arguments,
 their sum will be added to the total log density.
 
 
+## Increment log density with a change of variables adjustment
+
+
+A variant of the `target +=` statement described above is the
+`jacobian +=` statement. This can be used in the transformed parameters block
+or in functions ending with `_jacobian` to mimic the log Jacobian
+adjustments accrued by built-in variable transforms.
+
+Similarly to those implemented for the built-in transforms, these Jacobian adjustment
+may be skipped for optimization.
+
+For example, here is a program which re-creates the existing
+[`<upper=x>` transform](transforms.qmd#upper-bounded-scalar) on real numbers:
+
+```stan
+functions {
+  real upper_bound_jacobian(real x, real ub) {
+    jacobian += x;
+    return ub - exp(x);
+  }
+}
+data {
+  real ub;
+}
+parameters {
+  real b_raw;
+}
+transformed parameters {
+  real b = upper_bound_jacobian(b_raw, ub);
+}
```

Comment:
I think you want to go further and illustrate a positive-constrained distribution on `b`, such as

```stan
model {
  b ~ lognormal(0, 1);
}
```

Reply:
That is no longer "just" a recreation of the built in transform, so I've added it but commented out as an example

Reply:
Good point.  Then it probably makes more sense to add it as a separate comment to say that you can then use a distribution on the transformed parameter.